data/randomization/randomization-with-weather.ipynb

```diff
@@ -0,0 +1,732 @@
+{
```

Comment:
<p>So a good thing to do here (and maybe you already did it to select these criteria?) is to plot the distribution (histogram) of the rain amount and wind speed. Could you add a cell before getting the cell that computes the mean? This way we can easily see what the distributions look like and determine what is a good threshold for the "tail"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/uw-echospace/bat-detector-validation/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>